### PR TITLE
Change copyright statements

### PR DESCRIPTION
**Pull request by aurbroszniowski**

### CODE REVIEW

File: terracotta-kit/src/assemble/server/bin/start-tc-server.sh [hunk 1]
```diff
@@ -69,6 +69,7 @@ args="$@"
 start=true
 while "$start"
 do
+unset mod_args
 eval ${JAVA_COMMAND} -Xms2g -Xmx2g -XX:+HeapDumpOnOutOfMemoryError \
    -Dcom.sun.management.jmxremote \
```
Comment (guptakshit) [inline terracotta-kit/src/assemble/server/bin/start-tc-server.sh:74]: is this related to copyright?

File: common/src/test/resources/failover-priority-config.xml [hunk 2]
```diff
@@ -1,4 +1,8 @@
 <?xml version="1.0" encoding="UTF-8" ?>
+<!--
+    Copyright Terracotta, Inc.
+    Copyright Super iPaaS Integration LLC, an IBM Company 2024
+-->
 <tc:tc-config xmlns:tc="http://www.terracotta.org/config"
```
Comment (guptakshit) [inline common/src/test/resources/failover-priority-config.xml:6]: Since all files have either TPL or Apache license, and TPL license would be later converted to Apache license, should this file also have Apache license?

**Comment by guptakshit** [comment]
Following files have dual license mentioned both TPL and
```
/*
 * All content copyright (c) 2003-2008 Terracotta, Inc., except as may otherwise be noted in a separate copyright
 * notice. All rights reserved.
 */
 ```
 
 should we keep only one?

[AddPredicate.java](https://github.com/Terracotta-OSS/dso/blob/main/common/src/main/java/com/tc/async/api/AddPredicate.java)
[StageManager.java](https://github.com/Terracotta-OSS/dso/blob/main/common/src/main/java/com/tc/async/api/StageManager.java)
[TCPProxy.java](https://github.com/Terracotta-OSS/dso/blob/main/dso-common/src/main/java/com/tc/net/proxy/TCPProxy.java)
[MockTCClass.java](https://github.com/Terracotta-OSS/dso/blob/main/toolkit-impl/src/test/java/com/terracotta/toolkit/mockl2/test/MockTCClass.java)
[MockTCObject.java](https://github.com/Terracotta-OSS/dso/blob/main/toolkit-impl/src/test/java/com/terracotta/toolkit/mockl2/test/MockTCObject.java)

File: toolkit-impl/src/test/java/com/terracotta/tooolkit/mockl2/ds/ToolkitListTest.java [hunk 3]
```diff
@@ -69,7 +69,7 @@ public void logicalInvoke(Object object, LogicalOperation method, Object[] param
         
       }
     });
-    toolkitMap.put("organzation", "softwareag");
+    toolkitMap.put("organization", "IBM");
```
Comment (mathieucarbou) [inline toolkit-impl/src/test/java/com/terracotta/tooolkit/mockl2/ds/ToolkitListTest.java:72]: `IBM`, or `Super iPaaS Integration LLC, an IBM Company` ?

**Review by mathieucarbou — COMMENTED** [review]
.

File: toolkit-impl/src/test/java/com/terracotta/tooolkit/mockl2/ds/ToolkitListTest.java (hunk 3 above)
Comment (aurbroszniowski) [reply toolkit-impl/src/test/java/com/terracotta/tooolkit/mockl2/ds/ToolkitListTest.java:72]: it's just a test, not a copyright

**Review by guptakshit — APPROVED** [review]
[AddPredicate.java](https://github.com/Terracotta-OSS/dso/blob/main/common/src/main/java/com/tc/async/api/AddPredicate.java) still has dual license

**Comment by mathieucarbou** [comment]
@aurbroszniowski : you have so conflicts, don't know if you saw it ?

File: common/src/test/java/com/tc/stats/CircularLossyQueueTest.java [hunk 4]
```diff
@@ -1,18 +1,18 @@
...
- * Software distributed under the License is distributed on an "AS IS" basis,
- * WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for
- * the specific language governing rights and limitations under the License.
+ *      http://www.apache.org/licenses/LICENSE-2.0
  *
- * The Covered Software is Terracotta Platform.
- *
- * The Initial Developer of the Covered Software is 
- *      Terracotta, Inc., a Software AG company
+ *  Unless required by applicable law or agreed to in writing, software
```
Comment (guptakshit) [inline common/src/test/java/com/tc/stats/CircularLossyQueueTest.java:11]: this file has dual license apache and `All content copyright Terracotta`

File: common/src/test/java/com/tc/util/Grep.java [hunk 5]
```diff
@@ -1,18 +1,18 @@
...
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
  *
- * The Covered Software is Terracotta Platform.
+ *      http://www.apache.org/licenses/LICENSE-2.0
  *
- * The Initial Developer of the Covered Software is 
- *      Terracotta, Inc., a Software AG company
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
```
Comment (guptakshit) [inline common/src/test/java/com/tc/util/Grep.java:12]: this file also has sun microsystems copyright `Copyright 2001-2002 Sun Microsystems, Inc. All Rights Reserved.`, earlier this file also had TPL but now should it have only single copyright or both apache and sun microsystems?

File: build-parent/pom.xml [hunk 6]
```diff
@@ -1,19 +1,19 @@
...
 
-The Covered Software is Terracotta Platform.
-
-The Initial Developer of the Covered Software is 
-     Terracotta, Inc., a Software AG company
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
```
Comment (guptakshit) [inline build-parent/pom.xml:16]: this file has the following block with TPL
```
  <licenses>
    <license>
      <name>Terracotta Public License</name>
      <url>http://www.terracotta.org/legal/terracotta-public-license</url>
    </license>
  </licenses>
```

File: system-tests-parent/pom.xml [hunk 7]
```diff
@@ -1,19 +1,19 @@
...
-Software distributed under the License is distributed on an "AS IS" basis,
-WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for
-the specific language governing rights and limitations under the License.
+       http://www.apache.org/licenses/LICENSE-2.0
 
-The Covered Software is Terracotta Platform.
-
-The Initial Developer of the Covered Software is
-     Terracotta, Inc., a Software AG company
+   Unless required by applicable law or agreed to in writing, software
```
Comment (guptakshit) [inline system-tests-parent/pom.xml:12]: this file has the following TPL block
```
  <licenses>
    <license>
      <name>Terracotta Public License</name>
      <url>http://www.terracotta.org/legal/terracotta-public-license</url>
    </license>
  </licenses>
```

File: toolkit-impl/src/main/java/com/terracotta/toolkit/abortable/ToolkitAbortableOperationException.java [hunk 8]
```diff
@@ -1,18 +1,18 @@
...
- * WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for
- * the specific language governing rights and limitations under the License.
+ *      http://www.apache.org/licenses/LICENSE-2.0
  *
- * The Covered Software is Terracotta Platform.
- *
- * The Initial Developer of the Covered Software is 
- *      Terracotta, Inc., a Software AG company
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
```
Comment (guptakshit) [inline toolkit-impl/src/main/java/com/terracotta/toolkit/abortable/ToolkitAbortableOperationException.java:12]: this file has dual license

**Review by guptakshit — CHANGES_REQUESTED** [review]
1. If this has only apahce license then it would require apache LICENSE file and a notice file.
2. https://github.com/Terracotta-OSS/dso/blob/b325194cb7a248d8df46933497965cbb7b609dc8/terracotta-kit/src/assemble/README.txt#L56 needs license update

**Comment by prasantakumar82** [comment]
Should not you add a license file to the repo?

**Comment by cljohnso** [comment]
This PR appears to be based on a commit _other_ than main/HEAD.  @devlg75 is going to fix this up.

File: common/src/test/java/com/tc/util/Grep.java [hunk 9]
```diff
@@ -1,18 +1,38 @@
+/*
```
[the remark this answers is not in the document]
Comment (cljohnso) [reply common/src/test/java/com/tc/util/Grep.java:1]: Corrections for _this_ file are being reviewed.  Will attach when final.

**Review by cljohnso — CHANGES_REQUESTED** [review]

File: common/src/test/java/com/tc/util/Grep.java (hunk 9 above)
[the remark this answers is not in the document]
Comment (cljohnso) [reply common/src/test/java/com/tc/util/Grep.java:1]: This patch will make the recommended changes to `Grep.java`.  Applies against the current version of the PR.
[Fixup_Grep_java_copyright_license.patch](https://github.com/user-attachments/files/16444443/Fixup_Grep_java_copyright_license.patch)

File: common/src/test/resources/failover-priority-config.xml (hunk 2 above)
Comment (devlg75) [reply common/src/test/resources/failover-priority-config.xml]: this should be apache license, I have added the same.

**Comment by devlg75** [comment]
> Following files have dual license mentioned both TPL and
> 
> ```
> /*
>  * All content copyright (c) 2003-2008 Terracotta, Inc., except as may otherwise be noted in a separate copyright
>  * notice. All rights reserved.
>  */
> ```
> 
> should we keep only one?
> 
> [AddPredicate.java](https://github.com/Terracotta-OSS/dso/blob/main/common/src/main/java/com/tc/async/api/AddPredicate.java) [StageManager.java](https://github.com/Terracotta-OSS/dso/blob/main/common/src/main/java/com/tc/async/api/StageManager.java) [TCPProxy.java](https://github.com/Terracotta-OSS/dso/blob/main/dso-common/src/main/java/com/tc/net/proxy/TCPProxy.java) [MockTCClass.java](https://github.com/Terracotta-OSS/dso/blob/main/toolkit-impl/src/test/java/com/terracotta/toolkit/mockl2/test/MockTCClass.java) [MockTCObject.java](https://github.com/Terracotta-OSS/dso/blob/main/toolkit-impl/src/test/java/com/terracotta/toolkit/mockl2/test/MockTCObject.java)

need to keep only apache license, fixed.

**Comment by devlg75** [comment]
> 1. If this repository has only apache license then it would require apache LICENSE file and a NOTICE file.
> 
> 2. https://github.com/Terracotta-OSS/dso/blob/b325194cb7a248d8df46933497965cbb7b609dc8/terracotta-kit/src/assemble/README.txt#L56
>     needs license update?
>    does terracotta-kit/src/assemble require updates? like https://github.com/Terracotta-OSS/dso/blob/7593ad7275b5d4db8f2520afb9410245d6312c32/terracotta-kit/src/assemble/legal/Legal_Notice.txt#L14

it's necessary, i have added both.

File: common/src/main/java/com/tc/async/api/AddPredicate.java [hunk 10]
```diff
@@ -1,18 +1,18 @@
...
+ *      http://www.apache.org/licenses/LICENSE-2.0
  *
- * The Initial Developer of the Covered Software is 
- *      Terracotta, Inc., a Software AG company
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
  */
```
[the remark this answers is not in the document]
Comment (devlg75) [reply common/src/main/java/com/tc/async/api/AddPredicate.java]: fixed

File: management-agent/management-tsa-v2/src/test/java/com/terracotta/management/resource/services/AllEventsServiceTest.java [hunk 11]
```diff
@@ -1,18 +1,18 @@
...
+ *      http://www.apache.org/licenses/LICENSE-2.0
  *
- * The Covered Software is Terracotta Platform.
- *
- * The Initial Developer of the Covered Software is 
- *      Terracotta, Inc., a Software AG company
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
```
[the remark this answers is not in the document]
Comment (devlg75) [reply management-agent/management-tsa-v2/src/test/java/com/terracotta/management/resource/services/AllEventsServiceTest.java]: fixed

File: common/src/test/java/com/tc/stats/CircularLossyQueueTest.java (hunk 4 above)
Comment (devlg75) [reply common/src/test/java/com/tc/stats/CircularLossyQueueTest.java]: fixed

File: toolkit-impl/src/main/java/com/terracotta/toolkit/abortable/ToolkitAbortableOperationException.java (hunk 8 above)
Comment (devlg75) [reply toolkit-impl/src/main/java/com/terracotta/toolkit/abortable/ToolkitAbortableOperationException.java]: fixed

File: common/src/test/java/com/tc/util/Grep.java (hunk 5 above)
Comment (devlg75) [reply common/src/test/java/com/tc/util/Grep.java]: patch applied.

File: build-parent/pom.xml (hunk 6 above)
Comment (devlg75) [reply build-parent/pom.xml:16]: i have changed  it to
`  <licenses>
    <license>
      <name>Apache Public License</name>
      <url>http://www.apache.org/licenses/LICENSE-2.0</url>
    </license>
  </licenses>`

File: system-tests-parent/pom.xml (hunk 7 above)
Comment (devlg75) [reply system-tests-parent/pom.xml:12]: fixed.

File: common/src/test/java/com/tc/util/Grep.java (hunk 9 above)
[the remark this answers is not in the document]
Comment (devlg75) [reply common/src/test/java/com/tc/util/Grep.java:1]: patch applied.

File: terracotta-kit/src/assemble/distribution.xml [hunk 12]
```diff
@@ -1,21 +1,37 @@
...
+       WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+       See the License for the specific language governing permissions and
+       limitations under the License.
 
-    Software distributed under the License is distributed on an "AS IS" basis,
-    WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for
-    the specific language governing rights and limitations under the License.
+-->
+
+<!--
```
[the remark this answers is not in the document]
Comment (devlg75) [reply terracotta-kit/src/assemble/distribution.xml]: removed.

File: terracotta-samples/src/assemble/distribution.xml [hunk 13]
```diff
@@ -1,21 +1,37 @@
...
+       WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+       See the License for the specific language governing permissions and
+       limitations under the License.
 
-    Software distributed under the License is distributed on an "AS IS" basis,
-    WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for
-    the specific language governing rights and limitations under the License.
+-->
+
+<!--
```
[the remark this answers is not in the document]
Comment (devlg75) [reply terracotta-samples/src/assemble/distribution.xml]: removed.

File: build-parent/pom.xml [hunk 14]
```diff
@@ -31,8 +31,8 @@ The Initial Developer of the Covered Software is
 
   <licenses>
     <license>
-      <name>Terracotta Public License</name>
-      <url>http://www.terracotta.org/legal/terracotta-public-license</url>
+      <name>Apache Public License</name>
```
Comment (cljohnso) [inline build-parent/pom.xml:34]: The canonical reference to the Apache License v2 is:

```xml
<licenses>
  <license>
    <name>The Apache License, Version 2.0</name>
    <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
  </license>
</licenses>
```

File: system-tests-parent/pom.xml [hunk 15]
```diff
@@ -32,8 +32,8 @@ The Initial Developer of the Covered Software is
 
   <licenses>
     <license>
```
Comment (cljohnso) [inline system-tests-parent/pom.xml:34]: The canonical reference to the Apache License v2 is:

```xml
<licenses>
  <license>
    <name>The Apache License, Version 2.0</name>
    <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
  </license>
</licenses>
```

File: terracotta-kit/src/assemble/legal/Legal_Notice.txt [hunk 16]
```diff
@@ -11,4 +11,4 @@ Third Party Declaration:
 This software may include portions of third-party products. For third-party copyright notices, license terms, additional rights or restrictions, please refer to "License Texts, Copyright Notices and Disclaimers of Third Party Products". For certain specific third-party license restrictions, please refer to section E of the Legal Notices available under "License Terms and Conditions for Use of Software AG Products / Copyright and Trademark Notices of Software AG Products". These documents are part of the product documentation, located at http://softwareag.com/licenses and/or in the root installation directory of the licensed product(s).
 
 Confidentiality Disclaimer:
-Use, reproduction, transfer, publication or disclosure is prohibited except as specifically provided for in your License Agreement with Software AG. 
\ No newline at end of file
+Use, reproduction, transfer, publication oÏr disclosure is prohibited except as specifically provided for in your License Agreement with Software AG.
```
[the remark this answers is not in the document]
Comment (cljohnso) [reply terracotta-kit/src/assemble/legal/Legal_Notice.txt:14]: Someone made an errant keystroke in this file; it needs to be reverted.

File: terracotta-kit/src/assemble/server/bin/archive-tool.bat [hunk 17]
```diff
@@ -1,20 +1,37 @@
+@REM
```
[the remark this answers is not in the document]
Comment (cljohnso) [reply terracotta-kit/src/assemble/server/bin/archive-tool.bat:1]: Still need to fix this one ...

File: terracotta-kit/src/assemble/server/bin/server-stat.bat [hunk 18]
```diff
@@ -1,20 +1,37 @@
+@REM
```
[the remark this answers is not in the document]
Comment (cljohnso) [reply terracotta-kit/src/assemble/server/bin/server-stat.bat:1]: Still need to fix this one ...

File: terracotta-kit/src/assemble/server/bin/setenv.bat [hunk 19]
```diff
@@ -1,20 +1,37 @@
+@REM
```
[the remark this answers is not in the document]
Comment (cljohnso) [reply terracotta-kit/src/assemble/server/bin/setenv.bat:1]: Still need to fix this one...

File: terracotta-kit/src/assemble/server/bin/start-tc-server.bat [hunk 20]
```diff
@@ -1,20 +1,37 @@
+@REM
```
[the remark this answers is not in the document]
Comment (cljohnso) [reply terracotta-kit/src/assemble/server/bin/start-tc-server.bat:1]: Still need to fix this one ...

File: terracotta-kit/src/assemble/server/bin/stop-tc-server.bat [hunk 21]
```diff
@@ -1,20 +1,37 @@
+@REM
```
[the remark this answers is not in the document]
Comment (cljohnso) [reply terracotta-kit/src/assemble/server/bin/stop-tc-server.bat:1]: Still need to fix this one...

File: terracotta-kit/src/assemble/server/bin/version.bat [hunk 22]
```diff
@@ -1,20 +1,37 @@
+@REM
```
[the remark this answers is not in the document]
Comment (cljohnso) [reply terracotta-kit/src/assemble/server/bin/version.bat:1]: Still need to fix this one...

File: terracotta-samples/bin/buildcp.bat [hunk 23]
```diff
@@ -1,20 +1,37 @@
+@REM
```
[the remark this answers is not in the document]
Comment (cljohnso) [reply terracotta-samples/bin/buildcp.bat:1]: Still need to fix this one...

**Review by cljohnso — CHANGES_REQUESTED** [review]

**Review by cljohnso — APPROVED** [review]

File: common/src/test/java/com/tc/util/TimeMovesForwardTest.java [hunk 24]
```diff
@@ -0,0 +1,62 @@
...
+ *    limitations under the License.
+ */
+package com.tc.util;
+
+import com.tc.test.TCTestCase;
+
+public class TimeMovesForwardTest extends TCTestCase {
+
+  private static final long SAMPLE_DURATION = 60000;
+
```
Comment (guptakshit) [inline common/src/test/java/com/tc/util/TimeMovesForwardTest.java:24]: new test file

File: management-agent/management-tsa-impl-v1/src/main/java/com/terracotta/management/l1bridge/RemoteAgentService.java [hunk 25]
```diff
@@ -0,0 +1,110 @@
...
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.ExecutorService;
+
+/**
+ * @author Ludovic Orban
+ */
+public class RemoteAgentService implements AgentService {
+
+  private final L1MBeansSource l1MBeansSource;
```
Comment (guptakshit) [inline management-agent/management-tsa-impl-v1/src/main/java/com/terracotta/management/l1bridge/RemoteAgentService.java:44]: new management-agent-tsa-impl-v1 package is introduced

**Review by guptakshit — COMMENTED** [review]
I believe new/deleted things got introduced because originally the PR branch was created from a very old commit

**Review by guptakshit — APPROVED** [review]

File: server-script/pom.xml [hunk 26]
```diff
@@ -1,20 +1,20 @@
 <?xml version="1.0"?>
 <!-- 
-/* 
- * The contents of this file are subject to the Terracotta Public License Version
- * 2.0 (the "License"); You may not use this file except in compliance with the
- * License. You may obtain a copy of the License at 
+/**
```
Comment (cljohnso) [inline server-script/pom.xml:3]: This file has _odd_ comment content.  If it passes license validation, keep it but it's strange.

**Review by cljohnso — APPROVED** [review]
One minor gotcha.  If there are no other pending changes, I'm good to merge this.
